ouroboros-consensus/test-util/Test/Util/Orphans/Arbitrary.hs

```diff
@@ -19,25 +23,41 @@ import           Ouroboros.Network.Block (SlotNo (..))
 
 import           Ouroboros.Consensus.BlockchainTime
 import           Ouroboros.Consensus.ChainSyncClient (ClockSkew (..))
+import           Ouroboros.Consensus.Demo (defaultSecurityParam)
 import           Ouroboros.Consensus.Demo.Run
+import           Ouroboros.Consensus.Protocol.Abstract (maxRollbacks)
 import           Ouroboros.Consensus.Util.Random (Seed (..))
 
+import           Ouroboros.Storage.Common (EpochNo (..), EpochSize (..))
 import           Ouroboros.Storage.ImmutableDB.Layout
-import           Ouroboros.Storage.Common (EpochNo (..),
-                     EpochSize (..))
 
+import           Test.Dynamic.Util (ConnectionPlan (..),
```

Comment:
I recently noticed (via CI failure) that this breaks the `test-storage` component, but I haven't yet though through how to fix it.

Reply:
That is probably because the modules from `test-util` are used (and exported) both in `test-consensus` and `test-storage`. Most likely this can be fixed by adding a missing (package) dependency to `test-storage`.

However, I think eventually, `ConnectionPlan` and related types/functions can just move to its own module in `test-consensus`.